templates/guides/quickstart-dhcp.md

```diff
@@ -206,14 +206,31 @@ module "bloxone_infra_host_aws" {
 }
 ````
 
+You will need the ID for the dhcp host to assign the subnet to the BloxOne Host. 
+To get the ID, you can use the [bloxone_dhcp_hosts](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/data-sources/dhcp_hosts) data source. 
+Add the following code to your main.tf:
+
+````terraform
+data "bloxone_dhcp_hosts" "this" {
+  filters = {
+    name = module.bloxone_infra_host_aws.host.display_name
+  }
+
+  retry_if_not_found  = true
+  depends_on          = [module.bloxone_infra_host_aws]
+}
+````
+The `retry_if_not_found` attribute is set to true to allow the data source to retry if the host is not found immediately. The `depends_on` attribute is used to ensure that the data source is read after the BloxOne Host is created.
```

Comment:
We should mention the retry timeout is 20 minutes unless configured.

Reply:
Updated the default values in the description of the fields. 

Yes, the depends_on is important, otherwise the "plan" (before apply) will try to fetch the data source, and will wait for 20 mins unnecessarily.